app/views/document_topics/edit.html.erb

```diff
@@ -1,24 +1,39 @@
 <% content_for :back_link, document_path(@document) %>
 <% content_for :title, t("document_topics.edit.title", title: @document.title_or_fallback) %>
 
-<% def unroll(topic) %>
+<% def unroll(topic_content_id) %>
   <% capture do %>
     <li>
-      <%= @topic_index[topic][:title] %>
+      <%= tag.input type: "checkbox",
+        id: topic_content_id,
```

Comment:
Not that it's likely to cause a problem but it'd be more conventional if this was prefixed by a string e.g. "topic-#{content_id}"

Reply:
Done - this was interesting!

Reply:
Done :-).